client/src/components/Modals/PitchFeedback/questionsTab.tsx

```diff
@@ -0,0 +1,56 @@
+import React, { ReactElement, FC, useState } from 'react';
+import { ModalProps, Select } from 'semantic-ui-react';
+import { IPitchFeedback } from 'ssw-common';
+import { toString } from 'lodash';
+
+import './styles.scss';
+
+interface QuestionsTabProps extends ModalProps {
+  feedbacks: IPitchFeedback[];
+}
+
+const QuestionsTab: FC<QuestionsTabProps> = ({ feedbacks }): ReactElement => {
+  const [selectedQuestion, setSelectedQuestion] = useState('firstQuestion');
+  const QuestionOptions = [
```

Comment:
You could define this outside of the component if you want and export it in case it is needed elsewhere for extensibility